tests/Rules/Node/ValidConstantFunction/ValidConstantFunctionRuleTest.twig

```diff
@@ -7,4 +7,6 @@
     {{ constant('ThisDoesNotExist::SomeKey') }}
     {{ constant(someVar) }}
     {{ constant('ThisDoesNotExist::class') }}
+    {{ constant(1) }}
```

Comment:
@ruudk you wrote in your initial PR https://github.com/VincentLanglet/Twig-CS-Fixer/pull/277/commits/bffffcbfe58362034e9cc2c0dc48452e4c876971
```
        $constant = $argument->getAttribute('value');
        if (!\is_string($constant)) {
            return $node;
        }
```

Is there valid case when the value is not a string or should we display an error for this case ?

Reply:
I did this because the return type was mixed according to psalm or PHPStan. Shouldn't be the case I think.

Reply:
So maybe beter to report an error in that case.